recipes/tabulatehelper/meta.yaml

```diff
@@ -0,0 +1,42 @@
+{% set name = "tabulatehelper" %}
+{% set version = "0.0.7" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  # url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
```

Comment:
What is this about?

Reply:
@beckermr I don't know either. The download link by standard url doesn't work. But should. Republishing package on pypi doesn't help. But pip installs correctly as it uses this very hashed url as I put in the recipe. Strange...

Reply:
@beckermr  I'm not sure if this hash url is permanent but it's stable enough to build a conda package.